web/Views/Shared/_Layout.cshtml

```diff
@@ -26,18 +26,27 @@
                         <li class="nav-item">
                             <a class="nav-link text-dark" asp-area="" asp-controller="Home" asp-action="Privacy">Privacy</a>
                         </li>
-                        <li class="nav-item" style="margin-left: auto;">
-                            @{
-                                if (User.Identity?.IsAuthenticated == true)
-                                {
-                                    <a href="/TODO/profile" class="nav-link">@(((BricksAndHeartsUser)User.Identity).GoogleEmail)</a>
-                                }
-                                else
+                    </ul>

```

Comment:
Could you add a screenshot of what this frontend change looks like?

Reply:
![image](https://user-images.githubusercontent.com/79792127/179549118-47525daf-1357-417a-a330-b21a5880b210.png)
Will look like this

Reply:
Worth checking mobile as well if you haven't already, can simulate with dev tools https://developer.chrome.com/docs/devtools/device-mode/#viewport

I think it was already a bit wonky so that's fine, but we should make sure it's usable on mobile. No need for a screenshot, but if it's terrible then fix it or make another ticket